README.md

```diff
@@ -215,46 +215,15 @@ Contributing
 ------------
 
 Rules for contributions can be found in [CONTRIBUTING.md](CONTRIBUTING.md)
-
-Authors
--------
-
-### Maintainers* and Core Developers
-
-- Benjamin Worpitz* (original author)
-- Dr. Sergei Bastrakov*
-- Kseniia Bastrakova
-- Dr. Andrea Bocci*
-- Dr. Antonio Di Pilato
-- Simeon Ehrig
-- Luca Ferragina
-- Bernhard Manfred Gruber*
-- Christian Kaever
-- Dr. Jeffrey Kelling
-- Dr. Stewart Martin-Haugh
-- Aurora Perego
-- Jan Stephan*
-- René Widera*
-- Dr. Jeffrey Young
-
-### Former Members, Contributions and Thanks
-
-- Dr. Michael Bussmann
-- Mat Colgrove
-- Valentin Gehrke
-- Dr. Axel Hübl
-- Maximilian Knespel
-- Jakob Krude
-- Alexander Matthes
-- Hauke Mewes
-- Phil Nash
-- Dr. Felice Pantaleo
-- Dr. David M. Rogers
-- Mutsuo Saito
-- Jonas Schenke
-- Daniel Vollmer
-- Dr. Jiří Vyskočil
-- Matthias Werner
-- Bert Wesarg
-- Malte Zacharias
-- Erik Zenker
+Thanks to all [active and former contributors](.zenodo.json).
+
+Maintainers
+-----------
+
+- [Benjamin Worpitz](https://github.com/BenjaminW3)
+- [Dr. Andrea Bocci](https://github.com/fwyzard)
+- [Simeon Ehrig](https://github.com/SimeonEhrig)
+- [Dr. Axel Hübl](https://github.com/ax3l)
+- [Bernhard Manfred Gruber](https://github.com/bernhardmgruber)
+- [Jan Stephan](https://github.com/j-stephan)
+- [René Widera](https://github.com/psychocoderHPC)
```

Comment:
What is the selector for the people on the list? It does not correspond to the list in `.zenodo.json` :)

Reply:
This is a list of all current maintainers, see the caption `Maintainers`. 

Now I checked if the maintainer list is linkable directly. I will remove the names and point directly to https://github.com/orgs/alpaka-group/teams/alpaka-maintainers than this will not become outdated.

Reply:
I think this is the best solution.